frontend/doc-site/05_Annotate-and-Analyze-Your-Data/5_5-Gene-Sets.mdx

```diff
@@ -1,6 +1,6 @@
 # Gene Sets
 
-Cellxgene Desktop enables cellxgene users to pre-load gene sets for exploration, and to create or modify gene sets while exploring data. This is particularly useful for recording cell type markers when annotating cells THIS SHOULD LINK TO THE DESKTOP ANNOTATION CELLS PAGE. This section explains how to format gene sets data for preloading of gene sets and where to find gene sets information after making changes in the explorer.
+Cellxgene Desktop enables cellxgene users to pre-load gene sets for exploration, and to create or modify gene sets while exploring data. This is particularly useful for recording cell type markers when [annotating cells](/docs/05__Annotate%20and%20Analyze%20Your%20Data/5_0__Annotate%20and%20Analyze%20Your%20Data). This section explains how to format gene sets data for preloading of gene sets and where to find gene sets information after making changes in the explorer.
 
 To read more about how to use gene sets in the cellxgene interface, please see Exploring Gene Expression. LINK TO THIS GITHUB PAGE IS NOT WORKING FOR ME
```

Comment:
Need to add a link here to gene sets documentation (could link to the gene sets tutorial section - https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/frontend/doc-site/05_Annotate-and-Analyze-Your-Data/5_5-Gene-Sets.mdx)

Reply:
and also removal of this text 'LINK TO THIS GITHUB PAGE IS NOT WORKING FOR ME'

Reply:
isn't that the page we're currently on?

Reply:
oh yes my bad - I'm not sure where that 'Exploring Gene Expression' is supposed to link to? might need to ask meg about this